s4e-web/src/app/views/map-view/state/sentinel-search/sentinel-search.service.spec.ts

```diff
@@ -134,24 +130,24 @@ describe('SentinelSearchResultService', () => {
     });
 
     it('should clear search results', () => {
-      store.set([createSentinelSearchResult(SentinelSearchFactory.build(), apiPrefixV1)]);
+      store.set([createSentinelSearchResult(SentinelSearchFactory.build(), environment.apiPrefixV1)]);
       service.search({});
       expect(query.getAll().length).toBe(0);
     });
   });
 
   it('openModalForResult', () => {
     const spy = spyOn(modalService, 'show');
-    const result = createSentinelSearchResult(SentinelSearchFactory.build(), apiPrefixV1);
+    const result = createSentinelSearchResult(SentinelSearchFactory.build(), environment.apiPrefixV1);
```

Comment:
Now that `environment.apiPrefixV1` is static (known on compile time) it would be better to remove it from `createSentinelSearchResult` arguments list, and import it there directly.